api/test/models/auth_token_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AuthTokenTest < ActiveSupport::TestCase
```

Comment:
Recommend switching your test framework to Rspec

https://medium.com/@amliving/my-rails-rspec-set-up-6451269847f9

In that post you don't need database_cleaner anymore